docs/content/en/contribute/_index.md

```diff
@@ -0,0 +1,288 @@
+
+1. **Fork** the [K8ssandra docs repo](https://github.com/k8ssandra/k8ssandra-operator.git) on GitHub. 
+1. Make your changes and send a pull request (PR).
+1. If you're not yet ready for a review, put the PR in Draft mode to indicate 
+  it's a work in progress. (**Don't** add the Hugo property 
+  "draft = true" to the page front matter.)
+1. Wait for the automated PR workflow to do some checks.
+1. Continue updating your doc and pushing your changes until you're happy with 
+  the content.
+1. When you're ready for a review, add a comment to the PR, and remove it from Draft mode.
+1. After the Pull Request is reviewed and merged it will be deployed automatically. There is usually a delay of 10 or more minutes between deployment and when the updates are online. 
+
+### Updating a single page
+
+If you've just spotted something you'd like to change while using the docs, K8ssandra.io has a shortcut for you:
+
+1. Click **Edit this page** in the top right hand corner of the page.
+1. If you don't already have an up to date fork of the project repo, you are prompted to get one - click **Fork this repository and propose changes** or **Update your Fork** to get an up to date version of the project to edit. The appropriate page in your fork is displayed in edit mode.
+1. Follow the rest of the [Deploying to K8ssandra.io](#deploying-to-k8ssandraio) process above to make and propose your changes.
+
+### Previewing your changes locally
+
+If you want to run your own local Hugo server to preview your changes as you work:
+
+1. Follow the instructions to install [Hugo](https://gohugo.io/) and any other tools you need. You'll need at least **Hugo version 0.45** (we recommend using the most recent available version), and it must be the **extended** version, which supports SCSS.
+1. Fork the [K8ssandra repo](https://github.com/k8ssandra/k8ssandra-operator) repo into your own project, then create a local copy using `git clone`.
+
+    ```bash
+    git clone https://github.com/k8ssandra/k8ssandra-operator.git
+    ```
+
+1. Run `hugo server` in the docs site root directory, such as your `~/github/k8ssandra-operator/docs` directory. By default your site will be available at http://localhost:1313/. Now that you're serving your site locally, Hugo will watch for changes to the content and automatically refresh your site.
+1. Continue with the usual GitHub workflow to edit files, commit them, push the
+  changes up to your fork, and create a pull request.
+
+### Creating an issue
+
+If you've found a problem in the docs, but you're not sure how to fix it yourself, please create an issue in the [k8ssandra-operator repo](https://github.com/k8ssandra/k8ssandra-operator/issues). You can also create an issue about a specific page by clicking the **Create Issue** button in the top right hand corner of the page.
+
+## Technical requirements
+* Go >= 1.18
+* kubectl >= 1.23
+* kustomize >= 4.5.7 
```

Comment:
good point 👍 I'll add a `< 5.0.0` for now.
I'm not crazy about putting versions in documentation because that's something we always forget to update (or don't know we need to) when making changes in the code.